src/model/page.urls.ts

```diff
@@ -5,7 +5,11 @@ import { Templates } from "./template.paths";
  */
 
 const SEPARATOR: string = "/";
+export const OBJECTIONS: string = SEPARATOR + "strike-off-objections";
```

Comment:
We don't need this new constant definition - you can just use STRIKE_OFF_OBJECTIONS defined below.

Reply:
Using this.